lib/config/openssh/openssh.go

```diff
@@ -51,9 +51,9 @@ Host *.{{ $clusterName }} {{ $dot.ProxyHost }}
 Host *.{{ $clusterName }} !{{ $dot.ProxyHost }}
     Port 3022
     {{- if eq $dot.AppName "tsh" }}
-    ProxyCommand "{{ $dot.ExecutablePath }}" proxy ssh --cluster={{ $clusterName }} --proxy={{ $dot.ProxyHost }} %r@%h:%p
+    ProxyCommand "{{ $dot.ExecutablePath }}" proxy ssh --cluster={{ $clusterName }} --proxy={{ $dot.ProxyHost }} -l "%r" %h:%p
```

Comment:
Can we use the long option for better readability?
```suggestion
    ProxyCommand "{{ $dot.ExecutablePath }}" proxy ssh --cluster={{ $clusterName }} --proxy={{ $dot.ProxyHost }} --login "%r" %h:%p
```